app/views/saasbook/_topbar.html.erb

```diff
@@ -0,0 +1,88 @@
+                <i class="fas fa-bars"></i>
+                <i class="fas fa-arrow-left"></i>
+                <i class="fas fa-arrow-up"></i>
+            </button>
+            
+            
+            <div class="dropdown-buttons-trigger">
+                <button id="dropdown-buttons-trigger" class="btn btn-secondary topbarbtn" aria-label="Download this page"><i
+                        class="fas fa-download"></i></button>
+
+                <div class="dropdown-buttons">
+                    <!-- ipynb file if we had a myst markdown file -->
+                    
+                    <!-- Download raw file -->
+                    <a class="dropdown-button" href="#" download>
+                        <button type="button"
+                            class="btn btn-secondary topbarbtn" title="Download source file" data-toggle="tooltip"
+                            data-placement="left">.html
+                        </button>
+                    </a>
+                    <!-- Download PDF via print -->
+                    <a class="dropdown-button">
+                        <button type="button" id="download-print" class="btn btn-secondary topbarbtn" title="Print to PDF"
+                            onClick="window.print()" data-toggle="tooltip" data-placement="left">.pdf
+                        </button>
+                    </a>
+                </div>
+            </div>
+
+            <!-- Source interaction buttons -->
+            <a class="annotation-button" ><button type="button" class="btn btn-secondary topbarbtn" data-toggle="tooltip"
+                    data-placement="bottom" title="Make annotation"><i
+                        class="fa fa-comment"></i></button></a>
+
+            <a class="highlighter-button"><button type="button" class="btn btn-secondary topbarbtn" data-toggle="tooltip"
+                    data-placement="bottom" title="Highlighter"><i
+                        class="fas fa-highlighter"></i></button></a>
+            <!-- Full screen (wrap in <a> to have style consistency -->
+
+            <a class="full-screen-button">
+                <button type="button" class="btn btn-secondary topbarbtn" data-toggle="tooltip"
+                    data-placement="bottom" onclick="toggleFullScreen()" aria-label="Fullscreen mode"
+                    title="Fullscreen mode">
+                    <i class="fas fa-expand"></i>
+                </button>
+            </a>
+
+
+
+            <!-- Launch buttons -->
+
+        </div>
+        <!-- user_github_omniauth_authorize_path -->
+        <div class="d-none d-md-block col-md-2 py-1 show">
+            <% if user_signed_in? %>
+                <a class="user-page-button" href= "user-page-path%>">
+                    <button type="submit" class="btn btn-default" title="User page" 
+                            data-toggle="tooltip" data-placement="bottom">
+                        <%= current_user.full_name %>

```

Comment:
If we do a migration to rename full_name to nickname, these lines will need to be changed